src/scripts/main.js

```diff
@@ -1,11 +1,22 @@
 'use strict';
 
 function waitFor(element, eventName) {
-  // write your code here
+  return new Promise((resolve, reject) => {
+    element.addEventListener(eventName, () => {
+      resolve(
+        `It was ${eventName} +on the element: ${element.nodeName}, id: ${element.id}`,
```

Comment:
There is a typo in the template literal. The '+' sign should not be there. It should be: `It was ${eventName} on the element: ${element.nodeName}, id: ${element.id}`.